lib/octopus_serverspec_extensions/type/octopus_deploy_worker_pool.rb

```diff
@@ -28,20 +34,44 @@ def initialize(serverUrl, apiKey, worker_pool_name)
       @worker_pool = get_worker_pool_via_api(serverUrl, apiKey, worker_pool_name)
     end
 
+    def in_space(space_name)
+      # allows us to tag .in_space() onto the end of the resource. as in
+      # describe octopus_account("account name").in_space("MyNewSpace") do
+      @spaceId = get_space_id?(space_name)
+      if @worker_pool_name.nil?
+        raise "'worker_pool_name' was not provided. Unable to connect to Octopus server to validate configuration."
```

Comment:
We've already connected to the server to get space id?

Reply:
As the previous type, we only connect at the last opportunity now